docs/ml/deploy/_index.md

```diff
@@ -50,12 +51,6 @@ Follow [these instructions](/registry/advanced/mlmodel-design/) to design your m
 For some models of the ML model service, like the [Triton ML model service](https://github.com/viamrobotics/viam-mlmodelservice-triton/tree/main/) for Jetson boards, you can configure the service to use either the available CPU or a dedicated GPU.
 {{< /alert >}}
 
-You can use an ML model service to deploy:
-
-- a model from [the registry](https://app.viam.com/registry)
-- a model trained outside the Viam platform that you have uploaded
```

Comment:
i just thought this was two repetitive having in docs 3 times, can put back

Reply:
good change